packages/cascara/src/ui/Form/Form.js

```diff
@@ -178,6 +180,10 @@ const Form = ({
     !isEditable ? false : isInitialEditing
   );
 
+  const isInvalidFormState = !dataDisplay && !isInitialEditing;
+
+  useDeveloperMessage(isInvalidFormState, WARNING_STRINGS.MISSING_STATE_FORM);
```

Comment:
```suggestion
  useDeveloperMessage(isEditingWithoutDataDisplay, WARNING_STRINGS.INVALID_EDITING_AND_DISPLAY);
```
I suspect this was just from copy paste. :) The warning key should be descriptive of what the state is. It still is talking about missing state... but the warning is actually not about state hook missing. It is actually about an invalid combination of props. Perhaps `MISSING_STATE_FORM` could be something like `INVALID_EDITING_AND_DISPLAY`?

Reply:
Yeah. I was thinking more in continue to follow the idea from the other component. I will change it.